v2/blockstore/readonly.go

```diff
@@ -364,8 +394,8 @@ func (b *ReadOnly) AllKeysChan(ctx context.Context) (<-chan cid.Cid, error) {
 			select {
 			case ch <- c:
 			case <-ctx.Done():
-				// TODO: log ctx error
-				return
+				errHandler(ctx.Err())
+				return // TODO: log ctx error
```

Comment:
I think all these error TODOs can be removed now; we allow the user to handle them however they want via the error handlers.

Reply:
Left them since we have a dedicated issue on logging captured in https://github.com/ipld/go-car/issues/112